gradient/cli/common.py

```diff
@@ -242,3 +242,14 @@ def _compose_object_key(self):
 class GradientDatasetOption(GradientObjectListOption):
     def __init__(self, param_decls, **kwargs):
         super(GradientDatasetOption, self).__init__("dataset", param_decls, **kwargs)
+
+
+def validate_comma_split_option(comma_option_value, option_value):
+    if comma_option_value or option_value:
+        if option_value:
+            option_value = list(option_value)
+        else:
+            option_value = list()
+        if comma_option_value:
+            option_value.extend(comma_option_value.replace(" , ", ",").replace(", ", ",").split(","))
```

Comment:
Using .strip() on each string would be cleaner and more reliable that replace().
Also: please move this outside of function execution (move to separate variable)

Reply:
`strip` will not work on this. `comma_option_value` is whole string with separated by comma values. `strip` will remove whitespace only on beggining or end of string

```
a = " test test test test test test test "
a.strip(" ")
'test test test test test test test'
```

And yes I will move it outside of function call.